projects/osano/accessible.css

```diff
@@ -0,0 +1,8 @@
+.osano-cm-button:focus {
+    outline: 0.2rem auto Highlight;
```

Comment:
where are you getting `Highlight` from?

Reply:
oh i found a reference to it here, https://css-tricks.com/copy-the-browsers-native-focus-styles/ can we add a link to this or some other documentation somewhere?